packages/jazz-tools/src/tools/subscribe/CoValueCoreSubscription.ts

```diff
@@ -41,6 +41,16 @@ export class CoValueCoreSubscription {
     this.initializeSubscription();
   }
 
+  rehydrate() {
+    if (!this.unsubscribed) {
+      return;
+    }
+
+    this.unsubscribed = false;
+    this.initializeSubscription();
+    this.unsubscribe();
```

Comment:
Will this work? on `loadCoValue` we're bailing out early if `this.unsubscribed` is true